src/Microsoft.AspNetCore.Mvc.Core/MvcOptions.cs

```diff
@@ -64,15 +64,22 @@ public MvcOptions()
                 _allowValidatingTopLevelNodes,
                 _inputFormatterExceptionPolicy,
                 _suppressBindingUndefinedValueToEnumType,
-                _enableGlobalRouting,
+                _enableEndpointRouting,
             };
         }
 
-        // REVIEW: Add documentation
+        [Obsolete("Use EnableEndpointRouting.")]
         public bool EnableGlobalRouting
         {
-            get => _enableGlobalRouting.Value;
-            set => _enableGlobalRouting.Value = value;
+            get => EnableEndpointRouting;
+            set => EnableEndpointRouting = value;
+        }
```

Comment:
It should be possible to just delete this right? I don't think it leaked out of this repo...

Reply:
Web hooks used it to get their tests passing

Reply:
ah gotcha!